madprops/serializers.py

```diff
@@ -26,11 +26,42 @@ def parent_obj_field(self):
             '{0} misses relation to parent model'.format(self.model))
 
 
-class NestedPropertySerializerOptions(PropertySerializerOptions):
-    """Meta class options for NestedPropertySerializer"""
-    def __init__(self, meta):
-        super(NestedPropertySerializerOptions, self).__init__(meta)
-        self.exclude = ('id', self.parent_obj_field)
+class ListToDictSerializer(ListSerializer):
+    """This is how the DRF 3.0 works. For many=True case it automatically
+    returns a "List" serializer instead of original one
+    (__new__ method is overriden). Thus, we need to teach it to work with
+    {<prop_name>: <prop_value>,...} dict instead of standard
+    [{'name': <prop_name>, 'value': <prop_value>}...].
+    """
+    default_error_messages = []
+
+    @property
+    def data(self):
+        return super(ListSerializer, self).data
+
+    def to_representation(self, value):
+        # Since this is ListSerializer, it makes everything a list. We need
+        # dict (<name>: <value>}, so convert it to dictionary.
+        result_list = super(
+            ListToDictSerializer, self).to_representation(value)
```

Comment:
why not just `dict(super(ListToDictSerializer, self).to_representation(value))` ?